### ISSUE TITLE
Could not find OneSignal-4.6.4.aar

### ISSUE THREAD

**Issue by astraube** [OneSignal/OneSignal-Android-SDK#1509]
**Description:**

Could not find OneSignal-4.6.4.aar

https://repo.maven.apache.org/maven2/com/onesignal/OneSignal/4.6.4/

**Stacktrace:**

```
* What went wrong:
Execution failed for task ':app:mergeDebugNativeLibs'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Failed to transform OneSignal-4.6.4.aar (com.onesignal:OneSignal:4.6.4) to match attributes {artifactType=android-jni, org.gradle.category=library, org.gradle.dependency.bundling=external, org.gradle.libraryelements=aar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Could not find OneSignal-4.6.4.aar (com.onesignal:OneSignal:4.6.4).
        Searched in the following locations:
            https://repo.maven.apache.org/maven2/com/onesignal/OneSignal/4.6.4/OneSignal-4.6.4.aar
```

**Comment by Muhammadwahab** [comment]
Yes the same issue occurred on my device working fine on yesterday I unable to build the app due to this issue

**Comment by vladikstyle** [comment]
yeah,same issue today.

**Comment by Muhammadwahab** [comment]
> yeah,same issue today.

Hi ,did you able to fix this issue ?

**Comment by vladikstyle** [comment]
> > yeah,same issue today.
> 
> Hi ,did you able to fix this issue ?

no, i think this link not aviable https://jcenter.bintray.com/com/onesignal/OneSignal/4.6.4/OneSignal-4.6.4.aar
that is reason of issue

**Comment by Muhammadwahab** [comment]
> > > yeah,same issue today.
> > 
> > 
> > Hi ,did you able to fix this issue ?
> 
> no, i think this link not aviable https://jcenter.bintray.com/com/onesignal/OneSignal/4.6.4/OneSignal-4.6.4.aar that is reason of issue

yeh might be solved by one signal so waiting for their response

**Comment by oktayagca** [comment]
Hi,
same issue today and i solved it by adding mavenCentral() to gradle file.

**Comment by vladikstyle** [comment]
> Hi, same issue today and i solved it by adding mavenCentral() to gradle file.

doesnt help me( Show code please, where you add mavenCentral()? at repositaries ?

**Comment by oktayagca** [comment]
![ss](https://user-images.githubusercontent.com/58532036/149089056-68cf5b74-2891-4fc6-9c98-552968a3b982.PNG)

**Comment by Muhammadwahab** [comment]
thanks it working now by adding maven central in all repositry section

**Comment by vladikstyle** [comment]
yeah , thank you brother

> ![ss](https://user-images.githubusercontent.com/58532036/149089056-68cf5b74-2891-4fc6-9c98-552968a3b982.PNG)

**Comment by shahabrauf18** [comment]
Please make sure you have mavenCentral() in project level gradle like in below order.

```
buildscript {

    repositories {
        google()
        mavenCentral()
        gradlePluginPortal()
        jcenter()
    }
}

allprojects {
    repositories {
        google()
        mavenCentral()
        jcenter()
    }
}

```

**Comment by JhonatanRamirez** [comment]
Hi friends, I have the same problem, but even though I add the mavenCentral () it doesn't work for me, does anyone have any idea that it might happen?
![image](https://user-images.githubusercontent.com/22950195/149150944-110b9f78-b3f4-4938-93c0-3acf54aebf6d.png)

**Comment by JhonatanRamirez** [comment]
Hi friends, I have the same problem, but even though I add the mavenCentral () it doesn't work for me, does anyone have any idea that it might happen?
![image](https://user-images.githubusercontent.com/22950195/149158377-6545611a-9eb4-454a-9121-5b9fa1fca2ed.png)

![image](https://user-images.githubusercontent.com/22950195/149158493-e8fff308-2037-4520-8c98-5834b0e8d9fd.png)

**Comment by k3nyfk** [comment]
mavenCentral() was already added but the issue was still present. 
Switched back to 4.6.3.
`implementation 'com.onesignal:OneSignal:4.6.3'`

**Comment by JhonatanRamirez** [comment]
@k3nyfk Thanks, this worked I had been looking for the error for a long time.

**Comment by rollinb** [comment]
I am having to downgrade to 4.6.3 as well. mavenCentral is already in our repository lists but is not fixing the problem.

**Comment by shahabrauf18** [comment]
Add dependency like below as they have described in their setup [guide](https://documentation.onesignal.com/docs/android-sdk-setup) . 

```
dependencies {
    implementation 'com.onesignal:OneSignal:[4.0.0, 4.99.99]'
}
```

**Comment by jkasten2** [comment]
There was a signing issue when [4.6.4](https://github.com/OneSignal/OneSignal-Android-SDK/releases/tag/4.6.4) was released which cause the .aar file to be omitted from Maven Central. This was fixed in [4.6.5](https://github.com/OneSignal/OneSignal-Android-SDK/releases/tag/4.6.5)

**Comment by astraube** [comment]
> 4.6.

I did the same thing. It looks like the 4.6.4 pipe doesn't generate the .aar artifact

**Comment by astraube** [comment]
> There was a signing issue when [4.6.4](https://github.com/OneSignal/OneSignal-Android-SDK/releases/tag/4.6.4) was released which cause the .aar file to be omitted from Maven Central. This was fixed in [4.6.5](https://github.com/OneSignal/OneSignal-Android-SDK/releases/tag/4.6.5)

Thank you very much, problem solved.

**Comment by NashidB** [comment]
> ![ss](https://user-images.githubusercontent.com/58532036/149089056-68cf5b74-2891-4fc6-9c98-552968a3b982.PNG)

fixed my build too
